pkg/up/up.go

```diff
@@ -337,7 +380,9 @@ func (u *upRunner) createServicesAndGetPodHostAliases() ([]v1.HostAlias, error)
 			u.initResourceObjectMeta(&service.ObjectMeta, app.nameEncoded, app.name)
 			_, err := u.k8sServiceClient.Create(service)
 			if err != nil {
-				return nil, err
+				if err.Error() != errors.New("services \""+service.Name+"\" already exists").Error() {
```

Comment:
For a string `s`, `errors.New(s).Error()` can be simplified to `s`

Reply:
There is probably a reason why serviceName is double quoted (special character may be backslash escaped), so this method of error parsing may break for some service names. Is there an official way to do error checking for k8s client-go? (e.g. cast to specific error and check an error code).

Reply:
We should change the log message if the service already exists ("service already exists")

Reply:
k8s apimachinery handles the "already existing error". Implemented the same in this case.